requirements-local-whisper.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements.txt
```

Comment:
I have never tried this but I guess it works like I imagine it does? :grin:

Reply:
Yes! When testing, it installed everything from the requirements.txt